public/app/plugins/datasource/loki/querybuilder/operationUtils.ts

```diff
@@ -295,10 +295,19 @@ export function addNestedQueryHandler(def: QueryBuilderOperationDef, query: Loki
 
 export function getLineFilterRenderer(operation: string, caseInsensitive?: boolean) {
   return function lineFilterRenderer(model: QueryBuilderOperation, def: QueryBuilderOperationDef, innerExpr: string) {
+    // Strip out backticks within the string as backticks are used to wrap each param in the output
+    const params =
+      model.params?.map((param) => {
+        if (typeof param === 'string' && param.includes('`')) {
+          return param.replaceAll('`', '');
+        }
+        return param;
+      }) ?? [];
```

Comment:
That wouldn't work for a param which is wrapped in quotes, right? Like for this query:
```
{app="test"} |= "foo ` bar"
```
the backtick would be removed, or am I missing something?

Reply:
this is only applied in the query builder, which is always wrapped in back-ticks, you can still craft queries like this in the code editor.

Reply:
That is unfortunately still an issue: 

https://github.com/grafana/grafana/assets/8092184/f5f87804-a1f6-4d72-b7fa-d120ddb4063f

Reply:
try that in main though:
<img width="729" alt="image" src="https://github.com/grafana/grafana/assets/109082771/3c7f1674-56cb-4394-9acf-da8e5816479f">
It will always break the query right now if any backtick is present in the querybuilder

Reply:
Hm, how do we proceed here? While I agree it doesn't break the query anymore, I think it also shouldn't change the query. A
```
|= "foo ` bar"
```
line filter is obviously not the same as the
```
|= "foo bar"
```
line filter.

Reply:
yeah I'm not sure, backticks can't be escaped in a loki query, so I guess we'd either need to change the quote from ` to " when the user wants to write a query containing a backtick?
I'd argue that this isn't a painpoint for users until we see some evidence otherwise, and it's not something we should invest much/any time in?

Reply:
I think this should do it, no?

```typescript
export function getLineFilterRenderer(operation: string, caseInsensitive?: boolean) {
  return function lineFilterRenderer(model: QueryBuilderOperation, def: QueryBuilderOperationDef, innerExpr: string) {
    // Strip out backticks within the string as backticks are used to wrap each param in the output
    const hasBackticks = model.params.some((param) => typeof param === 'string' && param.includes('`'));
    const delimiter = hasBackticks ? '"' : '`';
    let params;
    if (hasBackticks) {
      params = model.params.map((param) =>
        typeof param === 'string' ? escapeLabelValueInExactSelector(param) : param
      );
    }else{
      params = model.params;
    }
    if (caseInsensitive) {
      return `${innerExpr} ${operation} ${delimiter}(?i)${params.join(
        `${delimiter} or ${delimiter}(?i)`
      )}${delimiter}`;
    }
    return `${innerExpr} ${operation} ${delimiter}${params.join(`${delimiter} or ${delimiter}`)}${delimiter}`;
  };
}
```

Reply:
🤦 haha let's find out

Reply:
<img width="483" alt="image" src="https://github.com/grafana/grafana/assets/109082771/1bf6fdb0-3705-4dc0-ad79-b9823cfc90f6">

seems to work quite nicely! I'll pull this in and update the tests. Thanks Sven!

Reply:
Only weirdness is that the query editor uses the same delimiter for all lines within a single operation.
<img width="611" alt="image" src="https://github.com/grafana/grafana/assets/109082771/dc85de96-f491-41a8-8b4c-b0221555a055">
But it should be fine?

Reply:
Yea, I don't think that's a huge problem.